src/stores/AccountDetailStore.ts

```diff
@@ -7,33 +7,62 @@ import AppStore from './AppStore';
 import Transaction from '../models/Transaction';
 
 export default class AccountDetailStore {
+  private static GET_TX_INTERVAL_MS: number = 60000;
+
   @observable public activeTabIdx: number = 0;
   @observable.shallow public items: Transaction[] = [];
   @observable public pageNum: number = 0;
   @observable public pagesTotal?: number;
 
-  constructor(private app: AppStore) {}
-
   @computed public get hasMore(): boolean {
     return !!this.pagesTotal && (this.pagesTotal > this.pageNum + 1);
- }
+  }
+
+  private getTransactionsInterval?: NodeJS.Timer = undefined;
+
+  constructor(private app: AppStore) {}
 
   @action public async loadFromWallet() {
     this.items = await this.load();
   }
 
   @action public async loadMore() {
     const txs = await this.load(this.pageNum + 1);
-
+    this.pageNum = this.pageNum + 1;
```

Comment:
Move this above the load call and use this.pageNum so you don't have to increment in two places